web/tests/api-spec.js

```diff
@@ -4,13 +4,15 @@ describe('api service', function () {
         fixture.setBase('tests/api-description');
 
         inject(function ($http) {
-            const MOCK_PROJECT_SETTINGS = {
-                API_ROOT: 'localhost:8000'
+            const mockProjectConfig = {
+                getSettings:() => ({
+                    apiRoot: 'localhost:8000'
+                })
```

Comment:
I think this syntax is confusing. It's not immediately apparent that wrapping the object in parens stops the curly braces from being parsed as an arrow-function block, and that removing the parens will break it.

What about
```js
const settings = {
    apiRoot: 'blah'
};
const mockProjectConfig = {
    getSettings: () => settings
};
```